client.go

```diff
@@ -120,7 +120,7 @@ func verifyFile(c *cli.Context) error {
 	if len(c.Args().First()) == 0 {
 		printErrAndExit("Please give the 'msgFile'", 1)
 	}
-	dbg.SetDebugVisible(c.GlobalInt("debug"))
+	log.SetDebugVisible(c.GlobalInt("debug"))
```

Comment:
It's kind of confusing to have `log` and `debug` and `SetDebug`. What is `log` a debugging utility package? Or a logging framework?